### PR TITLE
Lambda python

### PR DESCRIPTION
**Pull request by MickaelAlliel**

### CODE REVIEW

**Comment by MickaelAlliel** [comment]
readme and example are ready.
Doesnt work consistently @orishavit do you have a solution or theres nothing to do?

File: aws-lambda-python/lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from rook import auto_start
```
Comment (LiranLast) [inline aws-lambda-python/lambda_function.py]: I'm pretty sure that's not the way to use Rookout under Lambda :(

File: aws-lambda-python/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Quickstart for Python + Agentless Rookout on AWS Lambda
+
+A sample application for using Python + Agentless Rookout on AWS Lambda
```
Comment (LiranLast) [inline aws-lambda-python/README.md]: Please add a couple of words and a link about what is "agentless"

File: aws-lambda-python/lambda_function.py (hunk 1 above)
Comment (MickaelAlliel) [reply aws-lambda-python/lambda_function.py]: https://github.com/Rookout/Rookout/pull/580/files
Just saw this PR from ori, guess there was something to add to make it work. I'll wait until this is merged and I test it with this example

**Pull request merged**